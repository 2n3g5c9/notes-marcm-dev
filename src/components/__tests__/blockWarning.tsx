import React from 'react'
import renderer from 'react-test-renderer'

import { BlockWarning } from '@components'

describe('BlockWarning', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<BlockWarning />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
