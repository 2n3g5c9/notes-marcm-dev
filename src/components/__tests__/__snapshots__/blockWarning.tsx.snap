// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockWarning renders correctly 1`] = `
<div
  className="bg-yellow-100 border-yellow-300 border-l-2 sm:border-l-4 font-sans inline-block my-2 px-4 py-3 relative text-primary text-xs sm:text-sm sm:w-auto w-full"
/>
`;
